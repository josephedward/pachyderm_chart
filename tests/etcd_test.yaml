templates:
  # - dash-deployment.yaml
  - etcd-deployment.yaml
  # - pachd-deployment.yaml
tests:
  # or you can snapshot the whole manifest
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}