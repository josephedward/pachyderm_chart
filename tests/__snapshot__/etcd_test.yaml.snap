manifest should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: pach_chart
        app.kubernetes.io/version: 0.5.48
        helm.sh/chart: pach_chart-0.1.0
      name: RELEASE-NAME-pach_chart
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: pach_chart
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: pach_chart
        spec:
          containers:
          - image: pachyderm/etcd:v3.3.5
            imagePullPolicy: IfNotPresent
            livenessProbe:
              httpGet:
                path: /
                port: etcd-http
            name: pach_chart-etcd
            ports:
            - containerPort: 8080
              name: etcd-http
              protocol: TCP
            readinessProbe:
              httpGet:
                path: /
                port: etcd-http
            resources: {}
            securityContext: {}
          securityContext: {}
          serviceAccountName: RELEASE-NAME-pach_chart
